.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+Please help potential reviewers to understand this pull request and speed up 
+the process by answering the following questions, where appropriate.
+
+# Why was this pull request created?
+
+# How is this achieved?
+
+# How to test this?
```

Comment:
Yes, that makes sense to me, so that QA know what to test.  CI very often does not test the change, because unfortunately hardly any of us (except maybe Mate) are in the habit of writing tests for our changes.